library/ZendService/Twitter/Response.php

```diff
@@ -152,4 +160,52 @@ public function toValue()
     {
         return $this->jsonBody;
     }
+
+    /**
+     * Retun the array of response rate info
+     *
+     * @return array
+     */
+    public function getRateLimit()
+    {
+        return $this->rateLimit;
+    }
+
+    /**
+     * Populates the object with info. This can possibly called from the
+     * constructor, or it can be called later.
+     *
+     */
+    public function populate($httpResponse = null)
+    {
+
+        if (is_null($httpResponse)) {
```

Comment:
`! $httpResponse`. Also, I would disallow replacing the wrapped http response at runtime, since this object already represents a twitter-specific response value, and values don't really change once they are assigned